### ISSUE TITLE
Identity - Journal vs Website

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#1685]
@mdlincoln rightly raised an identity issue in #1682 that we sometimes struggle conceptually with what Programming Historian is, as it takes bits and pieces from both scholarly journals and more flexible websites.

I thought it would be helpful to clarify that through a discussion.

I'd also like to suggest that we present ourselves outwardly as the publisher of 3 independent journals of methodology (not research journals), which each can include a mixture of original lessons and translations (that just happen to usually come from one of the other publications in our offering). I think this best serves our attempts to enter conversations with other scholarly publishers, to seek funding and support, to integrate with libraries (and open access funding support), etc.

I don't think it affects how we operate, but perhaps does affect how we should describe ourselves externally and internally.

Any repercussions or comments on this?

**Comment by drjwbaker** [comment]
My position: we are a publisher of three journals and we use https://github.com/programminghistorian/jekyll/issues/1682 as a tool to more tightly define (for us, our authors, and our audiences) what we think an 'article' is.

**Comment by acrymble** [comment]
If there isn't any controversy here then I'll go through our public pages and identify any places where that messaging isn't clear or consistent.

**Comment by drjwbaker** [comment]
Thanks @acrymble

**Comment by acrymble** [comment]
Pages that contain text that may need revising in line with this:

- Front page - refers to 'versions' rather than 'publications'
- About page could make distinct publications clearer. Description sounds like a website rather than a journal. Specifically perhaps an intro sentence (this is what we are) and revising the references to "Teams" in the History section at the end, to refer instead to "publications"
- Contribute page talks about "supported languages" and should probably instead talk about "publications"
- author guidelines opening paragraph talks about the English "version" intsead of publication.
- support us first paragraph talks about a 'project' rather than 'publication' (this page also doesn't italicize programming historian consistently, but that's another issue). Page also talks about the Editorial Board of the Programming Historian, which should probably actually say ProgHist Ltd.

**Comment by acrymble** [comment]
Unless I hear otherwise in the next couple of days I will seek to rectify the above pages, with minimal intervention required (so as not to put a big burden on translation).

**Comment by drjwbaker** [comment]
Thanks for leading on this @acrymble. A couple of things:

> Front page - refers to 'versions' rather than 'publications'
> author guidelines opening paragraph talks about the English "version" instead of publication.

Do you mean here where we try to make it clear that the EN publication came first? If so, there was substantial discussion about this before https://github.com/programminghistorian/jekyll/issues/1089#issuecomment-441440524 and a PR you withdrew because you didn't want to reopen it https://github.com/programminghistorian/jekyll/pull/1417. In short, we didn't want to change the name to *The Programming Historian in English*, but we also needed some text that explain why it wasn't called *The Programming Historian in English* (when logically given the names of the ES and FR publication, it should be). I'm not sure what the solution is here.

> support us first paragraph talks about a 'project' rather than 'publication' (this page also doesn't italicize programming historian consistently, but that's another issue).

'Project' is everywhere (see https://programminghistorian.org/en/about). We either global fix to publisher, or accept that PH is a project that runs three (soon four..) publications.

> Page also talks about the Editorial Board of the Programming Historian, which should probably actually say ProgHist Ltd.

These are correct. "The Editorial Board of the Programming Historian" rather than ProgHist Ltd:

- did start to look at financial sustainability in 2018
- has the power to reject sponsorship funds
- agreed not to accept certain types of sponsorship
- has donations and sponsorship managed by me.

In short, when we set up the company, it was always the intention that the Editorial Board have control, which they do via our governance document https://github.com/programminghistorian/jekyll/wiki/Programming-Historian-Governance (that is, unless anyone objects to what I'm doing to raise money, I assume my activities are authorised by the Editorial Board). 

That said, I agree it is a bit messy and the 'support us' page may need looking at again as we get used to the business (loath to change it now as we are still getting used to the business being part of what we do, and - like you - I don't want to trigger loads of mini translations!)

**Comment by acrymble** [comment]
Thanks @drjwbaker. In terms of the company and the publications and legal ownership/protection, my understanding is that the company has to "own" the publications to afford its members protection under limited liability. So ProgHist Ltd publishes 3 journals, but has an internal policy of not interfering directly in editorial control within those.

Is that not right? I suspect different views on that may be why @mdlincoln has raised the matter of confusion.

**Comment by drjwbaker** [comment]
Correct. I agree this is knotty.

**Comment by acrymble** [comment]
Ok. If you adopt the "who do I sue" approach to the text, then I think this is misleading:

> Funding & Ownership
> 
> The Programming Historian is an international volunteer-driven project whose financial activities are administered by ProgHist Limited, a Not for Profit Company Limited by Guarantee that is registered in England, Company Number 12192946. It is published by the Editorial Board of the Programming Historian.

It should say (I think, something like):

> Funding & Ownership
> 
> The Programming Historian publications are owned by ProgHist Limited, a Not for Profit Company Limited by Guarantee that is registered in England, Company Number 12192946. The journals' editorial activities are controlled by independent editorial boards.

**Comment by drjwbaker** [comment]
You might be right. Rewording that stuff is bigger than this ticket though, I suspect.

**Comment by acrymble** [comment]
I'll add it to discussion at the next team meeting.

**Comment by acrymble** [comment]
As this issue now affects the ability of the Portuguese Team to translate core pages (because of possible future changes), I'm going to suggest the following pages are updated now:

- About page could make distinct publications clearer. Description sounds like a website rather than a journal. Specifically perhaps an intro sentence (this is what we are) and revising the references to "Teams" in the History section at the end, to refer instead to "publications"
- Contribute page talks about "supported languages" and should probably instead talk about "publications"
- author guidelines opening paragraph talks about the English "version" intsead of publication.

And that the following remain open to future discussion:

- Front page - refers to 'versions' rather than 'publications'
- support us first paragraph talks about a 'project' rather than 'publication' (this page also doesn't italicize programming historian consistently, but that's another issue). Page also talks about the Editorial Board of the Programming Historian, which should probably actually say ProgHist Ltd.

**Comment by acrymble** [comment]
The pull request was merged, covering most of the little text issues raised here. There remains a wider issue to work on related to formalising the relationship between ProgHist Ltd and the various publications, but I've opened a new ticket for that: #1704 where I think this conversation can continue.

The outcomes of this thread are that we should endeavour to talk about "Publications" rather than "teams" or "versions" moving forward. This is to avoid ambiguity and to align our language to the wider sector.
